### ISSUE TITLE
css and meteor.js

### ISSUE THREAD

**Issue by markizs** [bootcards/bootcards#40]
As it seems that bootcards is using some serious magic to use only one css file from three included, it is not compatible with meteor.js, which compiles all css into one big file. Maybe you have some idea how I could make bootcards work properly with all three css files being there at the same time? 

Or maybe you can point at the piece of code that does css file selection? It is unlikely that it I could somehow create my own version of bootcards that does css inclusion differently, but I could at least check it out. 

I have also described this issue at the repo that tries to package bootcards for meteor. MasahiroMorita/mmorita-bootcardsjs#2

thanks in advance for any pointers!

**Comment by westy92** [comment]
According to my observations, the server is detecting what the client is using and then serves up just that corresponding CSS file.  I am using PHP, so I utilized [Mobile Detect](http://mobiledetect.net/).

Some example PHP code using Mobile Detect:

        require_once 'Mobile_Detect.php';
        $detect = new Mobile_Detect();
        
        if ($detect->isiOS()){
           // iOS CSS include
        }
        else if ($detect->isAndroidOS()) {
           // Android CSS include
        }
        else {
           // desktop CSS include
        }

**Comment by markleusink** [comment]
That's correct. In the Bootcards demo (http://demo.bootcards.org), the app figures out from the user agent which CSS file to serve. See https://github.com/bootcards/demoapp/blob/master/app.js (around lines 45 and 100).

The demo app uses Handlebars.js on a Node server to dynamically generate the HTML send to the client, but you can off course also load the required stylesheet using client side JavaScript:

     var head = document.getElementsByTagName('head')[0],
     link = document.createElement('link');
     link.type = 'text/css';
     link.rel = 'stylesheet';
     link.href = 'url_to_bootcards_css.css';
     head.appendChild(link);

**Comment by Adiel-Sharabi** [comment]
If you want to use it in meteor you should place the css files in the public folder - they will not combined and serve "as is". 
Then you can do something like this:
```
<template name="contacts">
  <head>
        <link href="{{platformCss}}" rel="stylesheet" type="text/css">
  </head>
.... <!-- all your cards code -->
</template> 
```
and in the helper you can detect the agent like this (using [ua-parser-js](https://atmospherejs.com/faisalman/ua-parser-js)) 
```
Template.contacts.helpers({
  platformCss :function(){
    if($.ua.os.name=="Android"){
      return "/bootcards-android.css";
    }else {
      return "/bootcards-desktop.css";
    }
  }
})
```

**Comment by martunta** [comment]
ah, but I did not want it to be platform dependent, rather a responsive web with regards of screen size. If it is small screen, regardless of platform, show the neat mobile menu, if it is bigger, then show normal desktop style.

anyway, I have dropped bootcards and replaced with simpler custom card implementation since opening this issue (partly also because of this issue), so if it is not a problem for me personally anymore.
